source/simulator/stokes_matrix_free.cc

```diff
@@ -1703,6 +1779,111 @@ namespace aspect
             level_cell_data[level].enable_newton_derivatives = false;
         }
     }
+
+    {
+      // Create active mesh tables to store the product of the pressure perturbation and
+      // the normalized gravity used in the free surface stabilization.
+      // Currently, mutilevel is not implemented yet, it may slow down the convergence.
+
+      // TODO: implement multilevel surface terms for the free surface stabilization.
+
+      active_cell_data.apply_stabilization_free_surface_faces = false;
+
+      if (sim.mesh_deformation
+          && !sim.mesh_deformation->get_free_surface_boundary_indicators().empty())
```

Comment:
This will save you line 1803 below:
```suggestion
      active_cell_data.apply_stabilization_free_surface_faces = sim.mesh_deformation
          && !sim.mesh_deformation->get_free_surface_boundary_indicators().empty();

      if (active_cell_data.apply_stabilization_free_surface_faces == true)
```